src/lang/errors.test.ts

```diff
@@ -48,3 +49,15 @@ describe('test kclErrToDiagnostic', () => {
     ])
   })
 })
+
+describe('test errFromErrWithOutputs', () => {
```

Comment:
If you plan on keeping this, maybe move it to `src/lang/wasm.test.ts` since that's the module where `errFromErrWithOutputs` is defined.

Reply:
okay moved!